client/my-sites/stats/stats-purchase/stats-purchase-personal.tsx

```diff
@@ -1,16 +1,25 @@
 /* eslint-disable jsx-a11y/anchor-is-valid */
 import { PricingSlider, RenderThumbFunction } from '@automattic/components';
 import { Button } from '@wordpress/components';
+import { useTranslate } from 'i18n-calypso';
+import { COMPONENT_CLASS_NAME } from './stats-purchase-wizard';
 
 interface PersonalPurchaseProps {
 	subscriptionValue: number;
 	setSubscriptionValue: ( value: number ) => number;
 }
 
+const AVERAGE_PRICE_INFO = '$6';
+
 const PersonalPurchase = ( { subscriptionValue, setSubscriptionValue }: PersonalPurchaseProps ) => {
-	// TODO: Add translate calls to copy.
+	const translate = useTranslate();
+
 	const sliderLabel = ( ( props, state ) => {
-		return <div { ...props }>${ state?.valueNow || subscriptionValue }/month</div>;
+		return (
+			<div { ...props }>
+				${ state?.valueNow || subscriptionValue }/{ translate( 'month' ) }
```

Comment:
In the design, we are using yearly prices. Probably @ederrengifo can confirm.

Reply:
oh sorry for the confusion here! We initially used yearly prices but after checking with marketing we decide to make it monthly. I can double check that though